### PR TITLE
Added a utility class that helps the scale managers handle resize eve…

### PR DESCRIPTION
**Pull request by AMalley**
…nts.

Tasks:
- [x] Documentation
- [x] Unit Tests
- [ ] Typescript Typings
- [x] `npm run build:full`

### CODE REVIEW

File: src/scale-manager/ResizeHelper.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      this.resizeTick();
+    }
+  }
+
+  /**
+   * For older browsers, specifically for IE11, starts a loop making sure resize events are fired.
+   * @memberof ResizeHelper
+   */
+  resizeTick() {
+    setTimeout(() => this.resizeTick(), 10);
```
Comment (chipbell4) [inline src/scale-manager/ResizeHelper.js]: Why not do this as a `setInterval` instead?

Comment (AMalley) [reply src/scale-manager/ResizeHelper.js]: Fixed

File: src/scale-manager/SafeScaleManager.spec.js [hunk 2]
```diff
@@ -45,6 +45,16 @@ describe('Scale Manager', () => {
     expect(sm.callback.callCount).to.equal(0);
   });
 
+  it('Should not call resize in IE 11 environemnt', () => {
+    sm.callback = Sinon.fake();
+
+    Sinon.replace(window, 'dispatchEvent', () => {});
+    sm.resizer.resize();
+    Sinon.restore();
```
Comment (chipbell4) [inline src/scale-manager/SafeScaleManager.spec.js]: Move this `restore` call to an `afterEach`. That way, if this test begins to fail the missing `restore` call will always happen keeping other tests from failing..

Comment (AMalley) [reply src/scale-manager/SafeScaleManager.spec.js]: Fixed

**Pull request merged**